sdk/servicebus/azure-servicebus/README.md

```diff
@@ -177,24 +248,51 @@ import os
 connstr = os.environ['SERVICE_BUS_CONN_STR']
 topic_name = os.environ['SERVICE_BUS_TOPIC_NAME']
 subscription_name = os.environ['SERVICE_BUS_SUBSCRIPTION_NAME']
-session_id = os.environ.get('SERVICE_BUS_SESSION_ID')
 
 with ServiceBusClient.from_connection_string(connstr) as client:
     with client.get_topic_sender(topic_name) as sender:
-        sender.send(Message("Session Enabled Message", session_id=session_id))
+        sender.send(Message("Data"))
 
     # If session_id is null here, will receive from the first available session.
-    with client.get_subscription_session_receiver(topic_name, subscription_name, session_id) as receiver:
+    with client.get_subscription_receiver(topic_name, subscription_name) as receiver:
+        for msg in receiver:
+            print(str(msg))
+```
+
+### Settle a message after receipt
+
+When receiving from a queue, you have multiple actions you can take on the messages you receive.
+
+> **NOTE**: You can only settle `ReceivedMessage` objects having been returned from `ServiceBusReceiver.receive()` and `ServiceBusReceive.__iter__()` in `ReceiveSettleMode.PeekLock` mode
```

Comment:
You're correct about defer; but then we fall back into the trap of how do we differentiate peek()?  (which is why I was being so explicit about the receive mode to the point about being over-specific)  That was the _primary_ point of confusion that motivated this sort of note, that folks thought peeked messages could be settled too.  Perhaps my addendum below will address that sufficiently, but I'll chew this over as I'm addressing your point, since your core concern is accurate.